sw/otbn/crypto/p384_internal_mult.s

```diff
@@ -232,7 +343,7 @@ scalar_mult_int_p384:
   bn.sid    x2, 160(x26)
 
   /* double-and-add loop with decreasing index */
-  loopi     448, 85
+  loop      x16, 92
```

Comment:
can we please add a check of x16 whether it is either 448 or 578 and through an error otherwise?